lib/loader.js

```diff
@@ -61,24 +63,14 @@ function injectContext(app, delegate) {
           }
 
           class ContextModelClass extends OriginModelClass {
-            constructor(opts) {
-              super(opts);
-              this[ORIGINAL_MODEL_CLASS_NAME] = property;
-            }
-
             // custom setter always execute before define [CTX] when new Instance(super(opts) calling), if custom setter requires ctx, it should not be undefined
             get ctx() {
               return ctx;
             }
-            get $originalModelName() {
-              return this[ORIGINAL_MODEL_CLASS_NAME];
```

Comment:
already accessible from `app.model[name]`, besides `$originalModelName` isn't used in actual code yet, hence removed.